libknet/compress_model.h

```diff
@@ -11,7 +11,14 @@
 
 #include "internals.h"
 
-#define KNET_COMPRESS_MODEL_ABI 1
+#define KNET_COMPRESS_MODEL_ABI            1
```

Comment:
good spot here... we are changing the internal module API/ABI by adding the new call to get the default value. this should become 2.